src/config/theme.js

```diff
@@ -297,4 +270,41 @@ export const theme = {
 
 const bodhiTheme = createMuiTheme(theme);
 
+bodhiTheme.overrides = {
+  ...bodhiTheme.overrides,
+  MuiButton: {
```

Comment:
move this back into `theme.overrides`. do it inline.

Reply:
I want to use 'breakpoints' in 'overrides', but I didn't find a better way.

Reply:
got it :+1: